scripts/test_mkn.py

```
@@ -390,78 +471,160 @@ def test_mkn_missing_service():
+    )
+    assert isinstance(args.exact, bool), (
+        "args.exact must be a boolean"
+    )
+
+    utests = unit_tests()
+    itests = integration_tests()
+    _validate_no_duplicate_names("unit tests", utests)
+    _validate_no_duplicate_names("integration tests", itests)
+
+    if len(args.filters) > 0:
+        if args.exact == True:
```

Ruff (0.15.21)

[error] Avoid equality comparisons to `True`; use `args.exact:` for truth checks

Replace with `args.exact`

(E712)